docs/topics/request-response.rst

```diff
@@ -247,6 +245,8 @@ Request objects
 Other functions related to requests
 -----------------------------------
 
+.. autofunction:: scrapy.http.request.NO_CALLBACK
```

Comment:
Moved here, because if provided under `Request` it would be interpreted as `scrapy.http.request.Request.NO_CALLBACK` despite the absolute import path, and links to it would not work.